src/pages/Home/Home.js

```diff
@@ -150,34 +150,57 @@ const Home = () => {
           retina_detect: true,
         }}
       />
-      <div className={styles['desc-container']}>
-        <div className={styles['desc']}>
-          <div className={styles['desc-1']}>
-            <span className={styles['text']}>
-              Hi <img className={styles['wave']} src={Hand} alt="" />, I'm{" "}
+      <div className={styles["desc-container"]}>
+        <div className={styles["desc"]}>
+          <div className={styles["desc-1"]}>
+            <span className={styles["text"]}>
+              Hi <img className={styles["wave"]} src={Hand} alt="" />, I'm{" "}
               <span>
                 {" "}
                 <b> John Doe</b>
               </span>
             </span>
-            <div className={styles['center']}>
+            <div className={styles["center"]}>
               <img src={dp_male} alt="" />
             </div>
           </div>
-          <div className={styles['typing']}>
-           <span className={styles['AnimateEffect']}><Output/></span> 
+          <div className={styles["typing"]}>
+            I'm into{" "}
+            <span className={styles["AnimateEffect"]}>
+              <TypeAnimation
+                cursor={true}
+                sequence={[
+                  "Designing",
+                  1000,
+                  "UI/UX",
+                  1000,
+                  "Cloud Computing",
+                  1000,
+                  "Web Development",
+                  1000,
+                  "Open Source",
```

Comment:
- This should be defined as a separate component that is imported into the home.js
- The words should be defined inside a list/array so that it is easier for non-programmers to change it later on

Reply:
Okay I'll make the changes

Reply:
I made the changes could you check now?